src/commands/say.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  CacheType,
+  ChatInputCommandInteraction,
+  EmbedBuilder,
+} from 'discord.js';
+
+export const sayCommand = async (
+  interaction: ChatInputCommandInteraction<CacheType>
+) => {
+  if (!interaction.guild || !interaction.channel) return;
+
+  const content = interaction.options.getString('content', true);
+  await interaction.deferReply({ ephemeral: true });
+  const message = await interaction.channel.send(content);
+  await interaction.editReply('I said what you said!');
+
+  if (typeof process.env.SAY_LOGS_CHANNEL === 'string') {
```

Comment:
Maybe do this so that it only really works for non-empty values?

```suggestion
  if (process.env.SAY_LOGS_CHANNEL) {
```